settings.gradle.kts

```diff
@@ -99,7 +99,7 @@ gitRepositories {
     uri.set("https://github.com/hashgraph/hedera-protobufs.git")
     // choose tag or branch of HAPI you would like to test with
     // This version needs to match tha HAPI version below in versionCatalogs
-    tag.set("v0.35.0")
+    tag.set("05111-use-pbj-for-hcs-state")
```

Comment:
is this going to 'develop'?

Reply:
yes .. it will be checking out a branch of protobufs repo to work with PBJ